examples/30_extended/custom_flow_tutorial.py

```diff
@@ -0,0 +1,183 @@
+"""
+================================
+Creating and Using a Custom Flow
+================================
+
+The most convenient way to create a flow for your machine learning workflow is to generate it
+automatically as described in <>. However, there are scenarios where this is not possible, such
```

Comment:
Maybe one sentence before this to explain why they need flows at the first place?